mailchimp_subscriber.py

```diff
@@ -81,23 +97,43 @@ def process_users(users, list_id, mc_user, mc_key):
     """Takes a list of emails and returns a list of those not subscribed
     to the MailChimp list"""
     mc_client = MailChimp(mc_user, mc_key)
-    for client in users.values:
+    for client in users.values():
         set_mailchimp_status(client, mc_client, list_id)
-        add_user(client, mc_client, list_id)
+
+    if (SEND_MC_EMAIL):
```

Comment:
At line 65, the config.default is used to read this configuration option. Should that be used consistently here?

Reply:
replace with config dict

Reply:
Does this still need to be replaced?